scripts/ci-install.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -eo pipefail
+
+DIR="${BASH_SOURCE%/*}"
+if [[ ! -d "$DIR" ]]; then DIR="$PWD"; fi
+
+cd "${DIR}/.."
+
+# create artifact directory
+mkdir artifacts
+# remove container first, if it happens to exist
+make -f Makefile.ci ci-remove-container || echo "Container doesn't exist. Continue."
+# now do install and copy artifact to host
+make -f Makefile.ci ci-artifact-copy
+# remove container
+make -f Makefile.ci ci-remove-container
+# display contents of artifact dir
+ls -al artifacts
+# upload artifact
+buildkite-agent artifact upload ./artifacts/* "s3://${BUILDKITE_S3_ARTIFACT_EXCHANGE_BUCKET}/${BUILDKITE_PIPELINE_NAME}/"
```

Comment:
Oh is this how we actually publish the artifact to the HashiCorp Provider Registry?

Reply:
No. This is just uploading it as a Buildkite artifact. I'm not entirely sure what the entire process is for publishing the provider to HashiCorp.